### PR TITLE
Fixes #28781 - Disable Auto attach on Host

### PR DESCRIPTION
**Pull request by parthaa**
This commit does a couple of things
1) It disables auto attach setup in Simple Content Access enabled orgs.
2) It sets it auto attach status to 'Not Applicable'
3) Changed contentAccessMode=== 'org_enviroment' to
   'simpleContentAccessEnabled' when Simple Content Access is enabled.

### CODE REVIEW

**Comment by sseelam2** [comment]
I am seeing the content access mode banner for hosts and host collections which are not simple content enabled.

**Comment by parthaa** [comment]
> I am seeing the content access mode banner for hosts and host collections which are not simple content enabled.

ah found the issue. good catch :) . Updated

File: engines/bastion_katello/test/subscriptions/content-access-mode-banner.directive.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ describe('Directive: contentAccessModeBanner', function() {
     ));
 
     beforeEach(module(function($provide) {
-        $provide.value('contentAccessMode', 'org_environment');
+        $provide.value('simContentAccessEnabled', true);
```
Comment (jturel) [inline engines/bastion_katello/test/subscriptions/content-access-mode-banner.directive.test.js]: 'simple' :)

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-subscriptions-modal.html [hunk 2]
```diff
@@ -7,19 +7,19 @@ <h4 data-block="modal-header" translate>
 
     <div class="row">
       <div class="col-sm-12">
-        <button type="button" class="btn btn-default" ng-click="autoAttach()" ng-disabled="table.numSelected > 0 || contentAccessMode === 'org_environment'">
+        <button type="button" class="btn btn-default" ng-click="autoAttach()" ng-disabled="table.numSelected > 0 || simpleContentAccessEnabled">
           <span translate>Auto-Attach</span>
         </button>
 
-        <p class="help-text" ng-show="table.numSelected === 0 && contentAccessMode != 'org_environment'">
+        <p class="help-text" ng-show="table.numSelected === 0 && !simpleContentAccessEnabled'">
```
[the remark this answers is not in the document]
Comment (parthaa) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-subscriptions-modal.html]: Good catch

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html [hunk 3]
```diff
@@ -82,12 +82,12 @@ <h4 translate>Subscriptions</h4>
         </dd>
 
         <dt translate>Auto-Attach</dt>
-        <dd bst-edit-checkbox="host.subscription_facet_attributes.autoheal"
+        <dd ng-if= "simpleContentAccessEnabled" translate> Not Applicable </dd>
+        <dd ng-if= "!simpleContentAccessEnabled" bst-edit-select="host.subscription_facet_attributes.autoheal"
```
[the remark this answers is not in the document]
Comment (parthaa) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html]: Hmm I don't see this behaviour. Can you pull and try again.

**Comment by sseelam2** [comment]
Tested the functionality and it is working as expected in both Host and Host collections.

File: engines/bastion/app/views/bastion/layouts/assets.html.erb [hunk 4]
```diff
@@ -18,7 +18,7 @@
     angular.module('Bastion.features').value('FeatureSettings', angular.fromJson(<%= SETTINGS[:features].nil? ? {} : SETTINGS[:features].to_json.html_safe %>));
     angular.module('Bastion').value('currentLocale', '<%= I18n.locale %>');
     angular.module('Bastion').value('CurrentOrganization', "<%= Organization.current.id if Organization.current %>");
-    angular.module('Bastion').value('contentAccessMode', "<%= Organization.current.try(:content_access_mode) if Organization.current %>");
+    angular.module('Bastion').value('simpleContentAccessEnabled', <%= Organization.current.simple_content_access? if Organization.current %>)
```
Comment (sseelam2) [inline engines/bastion/app/views/bastion/layouts/assets.html.erb]: An extra space in front of <%= Organization and a semi-colon is missing in this statement. Just to be consistent :)
Apart from this the PR looks good.

**Comment by sseelam2** [comment]
APJ :+1:

**Pull request merged**